### PR TITLE
Update TI.com ruleset

### LINKED ISSUE #16639: Add new ruleset for ti.com
Type: new ruleset
Domain: http://ti.com/

### PR DESCRIPTION
**Pull request by lgoenner**
Update for the disabled Texas Instruments ruleset

- Datasheets ([example](https://www.ti.com/lit/ug/spruer9d/spruer9d.pdf)) and tools ([example](https://www.ti.com/tool/pmp9754)) can be viewed normally via HTTPS, so no exclusion should be necessary
- Most of the subdomains covered in the old ruleset simply redirect to www.ti.com or are no longer use
- Access via Tor Browser seems to be blocked

Closes #16639

### CODE REVIEW

**Comment by J0WI** [comment]
Thanks for researching them!

We document nonfunctional/problematic in the ruleset file itself:
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

Subdomains that only redirect should also be added as target: 
https://www.eff.org/https-everywhere/faq#why-does-https-everywhere-include-rules-for-sites-like-paypal-that-already-require-https-on-all-their-pages

Targets that work neither with HTTP nor HTTPS can be removed.

**Comment by lgoenner** [comment]
Could someone take a look at this?

`ERROR src/chrome/content/rules/Texas-Instruments.xml: Fetch error: http://www.ti.com/ => https://www.ti.com/: (28, 'Operation timed out after 30001 milliseconds with 0 bytes received')
ERROR src/chrome/content/rules/Texas-Instruments.xml: Fetch error: http://focus.ti.com/ => https://focus.ti.com/: (28, 'Operation timed out after 30001 milliseconds with 0 bytes received')
ERROR src/chrome/content/rules/Texas-Instruments.xml: Fetch error: http://fedid.ext.ti.com/ => https://fedid.ext.ti.com/: (28, 'Operation timed out after 30001 milliseconds with 0 bytes received')
ERROR src/chrome/content/rules/Texas-Instruments.xml: Fetch error: http://investor.ti.com/ => https://investor.ti.com/: (28, 'Operation timed out after 30001 milliseconds with 0 bytes received')`

If I'm reading it right it says that the URLs

- https://www.ti.com
- https://focus.ti.com
- https://fedid.ext.ti.com
- https://investor.ti.com

don't work (timeout). Even though some of them redirect, they work fine for me with HTTPS. Not sure what the issue is here, especially since www.ti.com worked in the CI test earlier.

**Comment by J0WI** [comment]
They are hosted by Akamai and at times they are blocking our tests. You can add an exclusion here:
https://github.com/EFForg/https-everywhere/blob/master/utils/ruleset-whitelist.csv

File: src/chrome/content/rules/TI.com-falsemixed.xml [hunk 1]
```diff
@@ -1,31 +0,0 @@
...
-		<exclusion pattern="^http://www.ti.com/(?:general/docs/)?lit/" />
-
-			<!--	+ve:
-					-->
-			<test url="http://www.ti.com/general/docs/lit/getliterature.tsp?baseLiteratureNumber=spruer9&amp;fileType=pdf" />
-			<test url="http://www.ti.com/lit/ug/spruer9d/spruer9d.pdf" />
-
-
-	<rule from="^http://ti\.com/"
-		to="https://www.ti.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/TI.com-falsemixed.xml]: Please keep this rule and add `ti.com` as target (but keep the comment about the timeout).

**Comment by lgoenner** [comment]
I'll remove focus.ti.com as a target and move it to the comments, since the issue is not limited to the main page (see http://focus.ti.com/general/docs/traininghome.tsp vs https://focus.ti.com/general/docs/traininghome.tsp)

**Comment by lgoenner** [comment]
If there's anything else, let me know

**Pull request merged**